process_files.go

```diff
@@ -30,19 +30,15 @@ func getFilesPath(path []string, exclude []string) ([]string, error) {
 	for _, p := range path {
 		p = strings.TrimSpace(p)
 
-		fileInfo, err := os.Stat(p)
-		if err != nil {
-			return nil, err
-		}
-
+		fileInfo, _ := os.Stat(p)
```

Comment:
A comment why we ignore err can be useful:
// no need to check err on os.stat.
// if we put ./test/*.yml, it will fail and it's normal

Reply:
done